docs/designers-developers/developers/tutorials/metabox/add-meta-block.md

```diff
@@ -0,0 +1,104 @@
+
+### Create Meta Block
+
+Next you need to create a new block which will be used to display the field to the user. See the [Block Tutorial](../../../../../docs/designers-developers/developers/tutorials/block-tutorial/readme.md) for a deeper understanding of creating custom blocks.
+
+For this block, you will use the PlainText component, which is similar to an HTML input text field. You can see the very large list of available components in the [editor package repository](https://github.com/WordPress/gutenberg/tree/master/packages/editor/src/components).
+
+
+A couple of items to note:
+
+1. The `withInstanceId` function creates a unique per component instance needed for the PlainText component.
+
+2. By using `meta` as the source for the attributes, the loading and storing of post meta happens automatically.
+
+
+Replace the previous content in `myguten.js` with the following:
+
+```js
+var el = wp.element.createElement;
+var registerBlockType = wp.blocks.registerBlockType;
+var PlainText = wp.editor.PlainText;
```

Comment:
We could use the `wp.components.Text` component instead. It takes care of styling and labeling for us. It also feels more appropriate to promote the use of `wp.components` than `wp.editor` to pull components into your plugin.

Reply:
Thanks @nosolosw ! I had it on my list to simplify, the style part didn't look right. The TextControl also implements the `withInstanceId` which simplifies it also. 👍